ts/api/pagopa.ts

```diff
@@ -134,12 +134,80 @@ type GetWalletsUsingGETExtraT = MapResponseType<
   WalletListResponse
 >;
 
+/**
+ *
+ * sanitaze psp tags if it is an array with values (avoid no string value and duplicates)
+ * otheriwise set tags as an empty array
+ * @param w the wallet object
+ */
+const fixWalletPspTagsValues = (w: any) => {
+  if (Array.isArray(w.psp.tags)) {
+    // tslint:disable-next-line: readonly-array
+    const duplicates: string[] = [];
+    // tslint:disable-next-line: no-object-mutation
+    w.psp = {
+      ...w.psp,
+      tags: w.psp.tags.filter((item: any) => {
+        if (typeof item === "string" && duplicates.indexOf(item) === -1) {
+          duplicates.push(item);
+          return true;
+        }
+        return false;
+      })
+    };
+  } else {
+    // tslint:disable-next-line: no-object-mutation
+    w.psp = { ...w.psp, tags: [] };
+  }
+  return w;
+};
+
+/**
+ * TODO: temporary patch. Remove this patch once SIA has fixed the spec.
+ * @see https://www.pivotaltracker.com/story/show/166665367
+ * This patch is due because 'tags' field (an array of strings) in psp objects
+ * often contains mixed (and duplicated too) values
+ * e.g tags = ["value1",null,null]
+ * Psp codec fails decoding 'tags' having these values, so this getPatchedWalletsUsingGETDecoder alterates the
+ * payload just before the decoding phase making 'tags' an empty array
+ */
+const getPatchedWalletsUsingGETDecoder = <A, O>(type: t.Type<A, O>) => {
```

Comment:
remove unneeded curly braces + `return`

Reply:
also you don't need the `type` parameter anymore here, you can replace it with `WalletListResponse`